### PR TITLE
Fix bug and enable ESLint

### LINKED ISSUE #56: ReferenceError: nobt is not defined
https://sentry.io/nobtio/frontend/issues/346580123/

```
ReferenceError: nobt is not defined
  at None (/main.d8a4547b4cc57fe9443c.js:11:97592)
  at None (/main.d8a4547b4cc57fe9443c.js:25:352839)
  at None (/main.d8a4547b4cc57fe9443c.js:25:364332)
  at dispatch (/main.d8a4547b4cc57fe9443c.js:11:145542)
  at None (/main.d8a4547b4cc57fe9443c.js:11:97842)
```

### PR DESCRIPTION
**Pull request by thomaseizinger**
Bug #56 was caused by an undefined variable.

This PR fixes #56 and introduces ESlint as part of our build process in order to avoid future problems like that. For now, ESlint is configured to only check for undefined variables. From now on, we can gradually enable other features in order to get a more consistent code base concerning formatting and other stuff.

### CODE REVIEW

File: src/routes/App/modules/currentNobt/actions.js [hunk 1]
```diff
@@ -17,10 +17,12 @@ function fetchNobtStarted() {
 export function fetchNobtSucceeded(response) {
   return (dispatch) => {
 
+    let nobt = response.data;
```
Comment (thomaseizinger) [inline src/routes/App/modules/currentNobt/actions.js]: This is the actual bug fix! :)

Comment (duffleit) [reply src/routes/App/modules/currentNobt/actions.js]: Can you please explain this fix to me?

Comment (thomaseizinger) [reply src/routes/App/modules/currentNobt/actions.js]: Sure.
This bug was introduced by this commit: https://github.com/thomaseizinger/frontend/commit/fd1804d9391efb698e7cac84c465d33c484cc960#diff-f4d1551ff5b81a1379a2942e2fa18869

I renamed the parameter from nobt to response and forgot to rename the usages as well. Therefore, the variable was now undefined, hence the bug.

Comment (duffleit) [reply src/routes/App/modules/currentNobt/actions.js]: Still don't get it. In bot cases payload contains a "nobt"-key?

Comment (thomaseizinger) [reply src/routes/App/modules/currentNobt/actions.js]: Did you read the commit message of the linked commit? It explains the initial bug. The method was called with different parameters. Once with `response` and once with `response.data`. Of course, accessing `data` a 2nd time does not make sense, hence the initial error.
After fixing that (in the commit above), I forgot to rename a variable (needed in the last line of this action creator, check the whole file in order to see it).

Comment (duffleit) [reply src/routes/App/modules/currentNobt/actions.js]: ah I got it - `nobt.name` was the problem.

Comment (thomaseizinger) [reply src/routes/App/modules/currentNobt/actions.js]: Exactly :)

**Pull request merged**